packages/scheduler/scheduler.py

```
@@ -536,6 +557,31 @@ async def force_cleanup(self) -> dict[str, int]:
+                if (
+                    "run-once" in tags
+                    and run_count == 0
+                    and cron == "* * * * *"
+                    and _age_seconds(doc.get("created_at")) > _EVERY_MINUTE_ORPHAN_AGE_SEC
+                ):
+                    try:
+                        remove_result = self._store.remove(job_id)
+                        if inspect.isawaitable(remove_result):
+                            await remove_result
+                    except Exception:
+                        pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)